examples/drupal_loadtest.rs

```diff
@@ -81,19 +81,27 @@ fn main() {
 }
 
 /// View the front page.
-async fn drupal_loadtest_front_page(client: &mut GooseClient) {
-    let mut response = client.get("/").await;
+async fn drupal_loadtest_front_page(client: &GooseClient) {
+    let mut response = client.get("/", None).await;
 
     // Grab some static assets from the front page.
     match response.response {
         Ok(r) => match r.text().await {
             Ok(t) => {
                 let re = Regex::new(r#"src="(.*?)""#).unwrap();
+                // Collect copy of URLs to run them async
+                let mut urls = Vec::new();
+                let mut futures = Vec::new();
                 for url in re.captures_iter(&t) {
                     if url[1].contains("/misc") || url[1].contains("/themes") {
-                        let _response = client.set_request_name("static asset").get(&url[1]);
+                        urls.push(url[1].to_string());
                     }
                 }
+                for index in 0..urls.len() {
+                    futures.push(client.get(&urls[index], Some("static asset")));
```

Comment:
@LionsAd here's an example of naming a request -- we used to use `set_request_name()`, but this is no longer possible as `client` is immutable. Instead we have to pass in the name with the request.

I'm leaning toward adding `get()` and `get_named()`, `post()` and `post_named`, etc, as I don't like passing in `None` or `Some("foo")`. Instead we'd have `client.get("/path/no/name");` and `client.get_named("/named/path", "foo");`.

Reply:
In the current version of this PR, we call `get_named()` to name the request -- as these are helpers, it feels more ergonomic to not have to pass in `None` or `Some("foo")` when not naming or naming a request.